src/reducers/versions.spec.tsx

```diff
@@ -2208,4 +2210,92 @@ describe(__filename, () => {
       ).toEqual(true);
     });
   });
+
+  describe('getRelativeDiff', () => {
+    const file1 = 'file1.js';
+
+    const _getRelativeDiff = ({ ...params }) => {
+      return getRelativeDiff({
+        currentAnchor: '',
+        diff: createFakeDiffWithChanges([[]]),
```

Comment:
Since arrays are allowed to have zero items, you could make this:

```diff
diff --git a/src/reducers/versions.spec.tsx b/src/reducers/versions.spec.tsx
index f2e0643..ad19ef7 100644
--- a/src/reducers/versions.spec.tsx
+++ b/src/reducers/versions.spec.tsx
@@ -2217,7 +2217,7 @@ describe(__filename, () => {
     const _getRelativeDiff = ({ ...params }) => {
       return getRelativeDiff({
         currentAnchor: '',
-        diff: createFakeDiffWithChanges([[]]),
+        diff: createFakeDiffWithChanges([]),
         pathList: [fakeVersion.file.selected_file],
         version: createInternalVersion(fakeVersion),
         ...params,
```

I think that's a little more straightforward.